Cannot compile TypeScript hello world client/server

I was following the [quickstart document](https://grpc.io/docs/quickstart/node.html) to implement the hello world client server example in TypeScript.

I can compile and run the code fine if I have `noImplicitAny` set to `false` in `tsconfig.json`.  However, if I have it set to `true`, I get the following error when running `tsc -p .`:
```
node_modules/grpc/index.d.ts:2:55 - error TS7016: Could not find a declaration file for module 'protobufjs'. '.../grpc-example/node_modules/protobufjs/dist/protobuf.js' implicitly has an 'any' type.
  Try `npm install @types/protobufjs` if it exists or add a new declaration (.d.ts) file containing `declare module 'protobufjs';`

2   import { Message, Service as ProtobufService } from "protobufjs";
                                                        ~~~~~~~~~~~~
```
Also, running `npm install @types/protobufjs` does not resolve the issue.

### Environment
 - OS name, version and architecture: OS X Mojave
 - Node version: 8.9.1
 - Node installation method: nvm
 - Package name and version: @grpc/proto-loader@0.3.0, grpc@1.15.1

After using `npm install @types/protobufjs@5.0.31`, compilation reports the following errors:
```
> tsc -p .

node_modules/@grpc/proto-loader/build/src/index.d.ts:42:40 - error TS2694: Namespace 'ProtoBuf' has no exported member 'IParseOptions'.

42 export declare type Options = Protobuf.IParseOptions & Protobuf.IConversionOptions & {
                                          ~~~~~~~~~~~~~


node_modules/@grpc/proto-loader/build/src/index.d.ts:42:65 - error TS2694: Namespace 'ProtoBuf' has no exported member 'IConversionOptions'.

42 export declare type Options = Protobuf.IParseOptions & Protobuf.IConversionOptions & {
                                                                   ~~~~~~~~~~~~~~~~~~


node_modules/@types/protobufjs/index.d.ts:275:16 - error TS2304: Cannot find name 'XMLHttpRequest'.

275         XHR(): XMLHttpRequest;
                   ~~~~~~~~~~~~~~


src/client.ts:19:33 - error TS2339: Property 'Greeter' does not exist on type 'Message | GrpcObject | typeof Client'.
  Property 'Greeter' does not exist on type 'Message'.

19   const client = new helloProto.Greeter('localhost:50051',
                                   ~~~~~~~


src/server.ts:26:32 - error TS2339: Property 'Greeter' does not exist on type 'Message | GrpcObject | typeof Client'.
  Property 'Greeter' does not exist on type 'Message'.

26   server.addService(helloProto.Greeter.service, {
                                  ~~~~~~~
```

--- a/packages/grpc-native-core/package.json
+++ b/packages/grpc-native-core/package.json
@@ -29,6 +29,7 @@
     "node-pre-gyp"
   ],
   "dependencies": {
+    "@types/protobufjs": "^5.0.31",
     "lodash.camelcase": "^4.3.0",
     "lodash.clone": "^4.5.0",
     "nan": "^2.0.0",

